content/get-started/new-leads/index.md

```diff
@@ -4,13 +4,29 @@ title: Set up your own community
 
 <div class='right'><img src="/images/undraw-illustrations/mind_map.svg" alt="" width="240" /></div>
 
-There are many communities established within Galaxy. We call them [Special Interest Groups](/community/sig/). We have SIGs based for example on
-[**region**](/community/sig/#regional-communities) or [**domain of science**](/community/sig/#communities-of-practice) but
-if you feel like you just don't see your own or don't feel identified with them, you can create your own community!
 
-Below you'll find a suggested path to follow before you can start your own community, a list of Do's and Don'ts and also useful links you should probably check out.
+There are many communities established within Galaxy. We call them [Special Interest Groups](/community/sig/). We have SIGs based for example on:
+- their [**region**](/community/#regional-communities) around the world - these are regional communities,
+- their [**domain of science**](/community/#communities-of-practice) - these are communities of practices in different scientific domains,
+- their [**topics of interest**](https://galaxyproject.org/community/wg/), such as systems, back-end, user interface - these are working groups. 
 
-### Path 
+We encourage you to look through those communities and if you find one that it is appropriate for your needs, you can join and contribute to that community. However, if you don't find a community that fits your work, domain or region, or don't feel identified with any of them, you can create your own community of practice!
+
+Below you'll find suggested recipes to follow before, during and after you can start your own community, and a list of Do's and Don'ts and also useful links you should probably check out.
+
+### Community Onboarding Cookbook
```

Comment:
Should we maybe move that to the bottom of the page? "Get involved" is maybe what we want to highlight a bit more?